KS-13-03 phone directory three

DenisGubernatorov opened the pull request:
1) Компании добавляются в общий список
2) Выводятся названия только существующих компаний

DenisGubernatorov committed:
KS-13-03-phone-directory-three
1) Компании добавляются в общий список
2) Выводятся названия только существующих компаний

DenisGubernatorov committed:
KS-13-03-phone-directory-three
1) Перенос кода задачи в соответствущий файл

DenisGubernatorov committed:
KS-13-03-phone-directory-three
1) Переименование класса, что бы избежать конфликтов  другими задачами

ievetrovdev commented on src/main/kotlin/lesson13/lesson13_task1.kt:
Обрати внимание во вкладке с изменениями в этот ПР попал лишний файл (13-1 который не относится к этой задаче). Скорее всего ветка создалась не из мастера. Давай выпилим его (надо в этой ветке физически удалить файл из проекта, сделать коммит удаления и запушить. и отправить ПР снова на ревью).

DenisGubernatorov replied:
Ветка создавалась из мастера. Но в первом коммите я не в том файле написал код. В коммите 73e3fb12a1179494acd2d6c811aec668bec2705e я переносил код из одного файла в другой.

DenisGubernatorov replied:
Файл удалил

DenisGubernatorov committed:
KS-13-03-phone-directory-three
1)Удаление файла lesson13_task1.kt В котором первоначально ошибочно был написан код

ievetrovdev reviewed (approved):
Все верно, отличное решение!
